### ISSUE TITLE
Running Video input for L3 Cornertrack

### ISSUE THREAD

**Issue by Sriaparna** [Xilinx/Vitis_Libraries#146]
While trying to enable the video input in Makefile by providing the video input file and enabling the #define VIDEO_INPUT true in xf_corner_tracker_config.h and the testbench, getting the following errors.
![Screenshot from 2022-07-22 06-46-16](https://user-images.githubusercontent.com/69186697/180340736-14f280b4-1744-4bdd-ad66-94998de05a3c.png)
When trying to fix them using extern method to declare globally facing another issue.
![Screenshot from 2022-07-22 06-47-33](https://user-images.githubusercontent.com/69186697/180340794-6e6c52f1-3e55-4d71-bcea-93f6e7a604f0.png)
Is there any other method to invoke videos? Because for Cornerrtrack and meanshift , only image folders are getting iterated. 
Thanks in advance.

**Comment by vt-lib-support** [comment]
Hi @Sriaparna ,

We are looking into this error in cornerTracker function.

Do you see the same issue for meanshiftracking function when video input is enabled?

**Comment by Sriaparna** [comment]
No, getting the frame output using cout in testbench but not able to write video to view the output of meanshifttracking.

**Comment by vt-lib-support** [comment]
The video writing part has been removed to lighten the current host code. But, that version is still available in the 2019.2 version, which you may try:
https://github.com/Xilinx/Vitis_Libraries/blob/2019.2/vision/L1/examples/meanshifttracking/xf_mean_shift_tb.cpp

**Comment by Sriaparna** [comment]
I tried the CornerTrack by defining almost all the out of scope variables, it's running but getting the folln. error:
![Screenshot from 2022-07-26 19-19-02](https://user-images.githubusercontent.com/69186697/181021947-12a992c2-3f2c-462c-bd01-48077032f08b.png)
